backends/events/default/default-events.cpp

```diff
@@ -196,7 +197,7 @@ bool DefaultEventManager::pollEvent(Common::Event &event) {
 				PauseToken pt;
 				if (g_engine)
 					pt = g_engine->pauseEngine();
-				GUI::MessageDialog alert(_("Do you really want to quit?"), _("Quit"), _("Cancel"));
+				GUI::MessageDialog alert(_("Do you really want to quit?\nAny unsaved in-game progress will be lost."), _("Quit"), _("Cancel"));
```

Comment:
You might want to only add the `Any unsaved in-game progress will be lost.` part of the message if there is a game in progress. It doesn't really make sense to have it when quitting from the launcher.

For the other case when returning to the launcher, the check is not really needed, but you might want to split the message in the same way so that for translators the common part about losing game progress will only need to be translated once.

The `Any unsaved in-game progress` bit sounds a bit weird to me as well. Maybe just `Any unsaved game progress`? It would be good to know what a native English speaker thing though.

Reply:
> The Any unsaved in-game progress bit sounds a bit weird to me as well. Maybe just Any unsaved game progress? It would be good to know what a native English speaker thing though.

Personally, I'd recommend dropping the reference to in-game or game, and just use "Any unsaved progress ... ", since you're only going to see the message when exiting a game.

Reply:
> You might want to only add the Any unsaved in-game progress will be lost. part of the message if there is a game in progress. It doesn't really make sense to have it when quitting from the launcher.

The current default behaviour, if added manually as a global option, is to NOT display the confirmation dialog if the user selects Quit from the Launcher Menu (it will display if closing the Launcher window or using the keyboard), so should confirm_exit be applied as a game option rather than global?